### PR TITLE
Added a contact form

### PR DESCRIPTION
**Pull request by Ngufor-emmanuella**
In this project i built a desktop portfolio version webpage with a min width of 768px. grid and flex position elements are used to position elements in my webpage. Also on buttons hover was applied to them . And a contact form was added to finalise the mobile version portfolio. no complex code is done here.
## Built With

- HTML
- CSS
- Linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -228,26 +235,34 @@ <h3><img src="Resources/Images/Rectangle57.png" alt="Skills"> <span class="about
...
+            <form action="https://formspree.io/f/xqknwlkr" method="post" id="f1">
+                <input name="text" id="text1" type="text" maxlength="30" placeholder="Full name" required>
+                <input name="email" id="email1" type="email" placeholder="Email adress" required>
+                <textarea name="message" id="text_zone1" cols="30" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
+                <input type="submit" value="Get in touch" >
+            </form>
+            <form action="https://formspree.io/f/xqknwlkr" method="post" id="f2">
+                <span>
+                    <input name="text" id="text2" type="text" maxlength="30" placeholder="First name" required>
+                    <input name="text" id="text" type="text" maxlength="30" placeholder="Second name" required>
```
Comment (codepantha) [inline index.html:250]: - [ ] Kindly change the input color of your form input to match the Figma design
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️

## Hi team 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🎯 
✔️ Descriptive PR
✔️ No linter errors
✔️ Correct GitHub flow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ngufor-emmanuella** [comment]
thank already changed the color

**Review by codecaiine — APPROVED** [review]
Hi @Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good project implementation :heavy_check_mark: 
- Great UI design :heavy_check_mark: 
- Good request changes application :heavy_check_mark: 
- No linters errors :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**